threescale_api/resources.py

```diff
@@ -520,9 +520,12 @@ class OIDCConfigs(DefaultClient):
     def url(self) -> str:
         return self.parent.url + '/oidc_configuration'
 
-    def update(self, params: dict = None, **kwargs) -> 'DefaultResource':
+    def update(self, params: dict = None, **kwargs) -> requests.Response:
         return self.rest.patch(url=self.url, json=params, **kwargs)
 
+    def read(self, params: dict = None, **kwargs) -> requests.Response:
+        return self.rest.get(url=self.url, json=params, **kwargs)
```

Comment:
Why are you returning `request.Response`?

If the request fails, the client will throw the `ApiClientError`.

I think you should return the content of the response  (`json()`)